google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -766,6 +790,7 @@ def generations
         #     f.content_encoding = "deflate"
         #     f.content_language = "de"
         #     f.content_type = "application/json"
+        #     f.custom_time = DateTime.now
```

Comment:
Does this respect UTC?

Reply:
I'll see if I can improve the example

Reply:
Updated to what I hope is a better example (open to suggestions).

Yes, `DateTime` does include UTC offset, and `new` without time args returns instances with 0 offset.

```
irb(main):014:0> DateTime.new(2025, 12, 31)
=> #<DateTime: 2025-12-31T00:00:00+00:00 ((2461041j,0s,0n),+0s,2299161j)>
irb(main):015:0> DateTime.new(2025, 12, 31).new_offset(0)
=> #<DateTime: 2025-12-31T00:00:00+00:00 ((2461041j,0s,0n),+0s,2299161j)>
```